components/core/FollowButton/FollowButton.test.js

```diff
@@ -0,0 +1,57 @@
+import { fireEvent, stubbedRender } from '@/testUtils';
+import FollowButton from './FollowButton';
+
+describe('<FollowButton />', () => {
+  const mockOnClick = jest.fn();
```

Comment:
It took me a while to understand what mockOnClick referred to on line 44. Maybe rename mockOnClick to jestFn ? or maybe inline everything and don't use global variables? Not sure what the best way to do it.

Reply:
Would it be more helpful to leave some comments around some of the setup? This is a convention of mocking functions in tests, at least in React. Not sure what kind of conventions Vue uses for testing though.

Reply:
If that's convention that's fine by me!